pages/linux/mkvmerge.md

```diff
@@ -0,0 +1,16 @@
+# mkvmerge
+
+> Merge and extract multimedia streams
+> More information: <https://github.com/topics/mkvmerge>.
```

Comment:
```suggestion
> More information: <https://manned.org/mkvmerge>.
```

Reply:
I think the main website is harder to find and it have a better explanation about the command.
```suggestion
> More information: <https://mkvtoolnix.download/doc/mkvmerge.html>.
```

Reply:
I think we should prefer the official page here: <https://mkvtoolnix.download/doc/mkvmerge.html>.